docs/DeveloperGuide.adoc

```diff
@@ -475,23 +475,34 @@ is rather high. Implementing alternative 2 would be more tedious on the user's s
 // tag::list[]
 === List (Mainlist, Blacklist etc) mechanism
 
-List feature is a useful feature to view persons of a certain category. In the context of this project, master list contains all the contacts of the debt collection agency.
+Multiple List feature is a useful feature to view debtors of different categories. Currently these different lists include masterlist and blacklist.
```

Comment:
*Multiple list

Reply:
Having multiple lists is useful for debt collectors to view debtors of different categories.*

Reply:
Noted